app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -92,7 +93,16 @@ protected void onCreate(final Bundle savedInstanceState) {
 
         sortOnName();
 
-        this.serviceLocator.getBluetoothHandler().start();
+        try {
+            this.serviceLocator.getBluetoothHandler().start();
+        } catch (NoBluetoothException e) {
+            LOGGER.info("Bluetooth not available on device, hiding button");
+            final View v = this.findViewById(R.id.pairing_menu_bluetooth);
+            v.setVisibility(View.GONE);
+        } catch (IOException e) {
+            LOGGER.info("Bluetooth handler not started, most likely Bluetooth is not enabled");
```

Comment:
Could you show a dialog to inform the user about this and ask him if Bluetooth should be turned on?
This can help you: https://stackoverflow.com/questions/3806536/how-to-enable-disable-bluetooth-programmatically-in-android